CODEOWNERS

```diff
@@ -457,7 +457,7 @@ homeassistant/components/syncthru/* @nielstron
 homeassistant/components/synology_dsm/* @hacf-fr @Quentame @mib1185
 homeassistant/components/synology_srm/* @aerialls
 homeassistant/components/syslog/* @fabaff
-homeassistant/components/tado/* @michaelarnauts @bdraco
+homeassistant/components/tado/* @michaelarnauts @bdraco @noltari
```

Comment:
This needs to be added to the integration manifest file, and next run hassfest to update all other configuration files.

Reply:
Right, I totally forgot about that.